### ISSUE TITLE
Time of the container

### ISSUE THREAD

**Issue by gijswobben** [splunk/docker-splunk-legacy#30]
At the Splunk startup page I get this error message:

```
Warning: The time on the server differs significantly from this machine which may cause login problems and other errors.
```

Is there any way to fix this?

**Comment by halr9000** [comment]
Where is Docker running? I've heard of a time drift issue amen running Docker on OSX.

**Comment by gijswobben** [comment]
Docker for Windows 1.12.2

**Comment by michaelwilde** [comment]
@gijswobben is this still an open issue?

**Comment by gijswobben** [comment]
No, was fixed with docker 1.13
